backend/__fixtures__/secrets.js

```diff
@@ -260,6 +261,10 @@ const mocks = {
           const item = secrets.getShootSecret(namespace, name)
           return Promise.resolve(item)
         }
+        if (name === 'newest-secret') {
```

Comment:
We should not make assumptions on certain names. Only assumptions on k8s conventions or gardener conventions.